apps/wallet-mobile/src/kernel/i18n/locales/en-US.json

```
@@ -377,6 +377,11 @@
   "components.settings.applicationsettingsscreen.title": "Settings",
   "components.settings.applicationsettingsscreen.version": "Current version:",
   "components.settings.applicationsettingsscreen.walletType": "Wallet type",
+  "components.settings.applicationsettingsscreen.network.notice.title": "What are the test networks?",
+  "components.settings.applicationsettingsscreen.network.notice.message": "The test networks serve as a platform for the community and developers to test products and experiments without risking real funds on the mainnet.",
+  "components.settings.applicationsettingsscreen.network.notice.listTitle": "Key features of testnet coins:",
+  "components.settings.applicationsettingsscreen.network.notice.list": "  •  Have no real value.\n  •  Are separate from the mainnet.\n  •  Are separate from the mainnet.\n  •  Cannot be sent to mainnet wallets.\n  •  Are easily obtainable from Cardano faucets.",
```

twice separated from mainnet.
```suggestion
  "components.settings.applicationsettingsscreen.network.notice.list": "  •  Have no real value.\n  •  Are separate from the mainnet.\n •  Cannot be sent to mainnet wallets.\n  •  Are easily obtainable from Cardano faucets.",
```

---

done